.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -316,7 +316,7 @@ jobs:
               -Dpybind11_DIR=$(python${{ matrix.python_version }} -c "import pybind11; print(pybind11.get_cmake_dir())") \
               -DENABLE_LIGHTNING_KOKKOS=ON \
               -DENABLE_LAPACK=OFF \
-              -DLIGHTNING_GIT_TAG=476cac2e39f79c122b8751a77aad8139b39296db \
+              -DLIGHTNING_GIT_TAG=latest_release \
```

Comment:
This is still v0.36, is that intended?

Reply:
@erick-xanadu some thoughts on this?

Reply:
This PR will be merged after the lightning release